config/locales/help.en.yml

```diff
@@ -0,0 +1,737 @@
+en:  
+  new_help:
+    how_to:
+      title: How do I get started?
+      start_group_html: '<strong><a href="#starting-new-group">Start a group</a></strong> to collaborate with people on Loomio'
+      invite_people_html: '<strong><a href="#inviting-members">Invite people</a></strong> into your group'
+      start_discussion_html: '<strong><a href="#starting-discussions">Start a discussion</a></strong> on any topic'
+      make_proposal_html: '<strong><a href="#starting-proposals">Make a proposal</a></strong> to see how everyone feels about a particular course of action'
+      decide_together_html: '<strong><a href="#stating-positions">Decide together:</a></strong> anyone can agree, abstain, disagree, or block – so you can see how everyone feels about the proposal.'
+    contents:
+      title: Need more information?
+      getting_started:
+        title_html: '<a href="#getting-started">Getting started</a>'
+        starting_new_group_html: '<a href="#starting-new-group">When you want to start a new Loomio group</a>'
+        invited_into_group_html: '<a href="#invited-into-group">When you are invited into a Loomio group</a>'
+        joining_existing_group_html: '<a href="#joining-existing-group">When you want to join an existing Loomio group</a>'
+        single_sign_on_html: '<a href="#single-sign-on">Single sign-on</a>'
+      group_settings:
+        title_html: '<a href="#group-settings">Group settings</a>'
+        group_profile_html: '<a href="#group-profile">Group profile</a>'
+        group_privacy_html: '<a href="#group-privacy">Group privacy</a>'
+        group_permissions_html: '<a href="#group-permissions">Group permissions</a>'
+      coordination:
+        title_html: '<a href="#coordination">Coordinating your group</a>'
```

Comment:
I guess I don't see why we need to include the anchor tags and their href's in these translations... seems like a hurdle for translators that doesn't need to be there.

Why not put the anchors in the haml, and the translations in the yml file?

```
%a{href: '#group-settings'} = t :"group_settings.title_html"
```